app/models/shape_point.rb

```diff
@@ -0,0 +1,4 @@
+class ShapePoint < ActiveRecord::Base
+  belongs_to :shape
+  has_one :agency, through: :shape
```

Comment:
It's just so we can do `@agency.stop_points.destroy` in the importer. Looked a little better than joining through shapes within the importer